src/Kovan-DssSpell.sol

```diff
@@ -270,13 +469,22 @@ contract DssSpellAction is DssAction {
             cut: 99 * RAY / 100,
             step: 90 seconds,
             tolerance: 50 * RAY / 100,
-            clipKey: "MCD_CLIP_WBTC_A",
-            calcKey: "MCD_CLIP_CALC_WBTC_A",
-            flipKey: "MCD_FLIP_WBTC_A"
+            clipKey: "MCD_CLIP_AAVE_A",
+            calcKey: "MCD_CLIP_CALC_AAVE_A",
+            flipKey: "MCD_FLIP_AAVE_A"
         }));
 
+        // --------------------------------- Fix wrong PSM keys (only for kovan) ---------------------------------
+        address flipperPSM = DssExecLib.getChangelogAddress("MCD_FLIP_USDC_PSM");
```

Comment:
Why use a local var for this but not the others?

Reply:
Ahh because it was being used twice, but have removed the clipper part as stables aren't going this week at the end. Will change it

Reply:
Done